runtime/jcl/common/jdk_internal_misc_ScopedMemoryAccess.cpp

```diff
@@ -34,11 +36,58 @@ Java_jdk_internal_misc_ScopedMemoryAccess_registerNatives(JNIEnv *env, jclass cl
 {
 }
 
+static UDATA
+closeScope0FrameWalkFunction(J9VMThread *vmThread, J9StackWalkState *walkState)
+{
+	if (*(UDATA *)walkState->userData2) { /* scope has been found */
```

Comment:
I suggest using `if (TRUE == *(UDATA *)walkState->userData2)` instead of implicit if check